projects/angular/src/forms/styles/_containers.clarity.scss

```diff
@@ -136,7 +151,11 @@
   }
 
   .clr-validate-icon {
-    @include mixins.min-equilateral(forms-variables.$clr-forms-icon-size);
+    @include mixins.equilateral-var(
```

Comment:
Different equilateral mixin should be used at this place, probably `min-equilateral-var` should be created to handle the changes